File: Programmers/Level1/[1차]비밀지도/[1차]비밀지도.py
def solution(n, arr1, arr2):
    answer = []
    
    for a1, a2 in zip(arr1, arr2):
        tmp = bin(a1 | a2)[2:]
        tmp = '0' * (n - len(tmp)) + tmp if len(tmp) < n else tmp
        answer.append(tmp.replace('0', ' ').replace('1', '#'))
    
    return answer

File: Programmers/Level1/[1차]비밀지도/[1차]비밀지도.js
function solution(n, arr1, arr2) {
    const answer = [];

    for (let i = 0; i < n; i++) {
        let tmp = (arr1[i] | arr2[i]).toString(2);
        tmp = tmp.length < n ? "0".repeat(n - tmp.length) + tmp : tmp;
        answer.push(tmp.replace(/0/g, " ").replace(/1/g, "#"));
    }

    return answer;
}
